man/systemd.network.xml

```diff
@@ -2359,6 +2359,32 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TokenBufferFilterLatencySec=</varname></term>
+        <listitem>
+          <para>Specifies the latency parameter, which specifies the maximum amount of time a
+          packet can sit in the TBF. The latter calculation takes into account the size of the bucket,
```

Comment:
Please expand "TBF".

I don't understand the second sentence. The latency is a time. What does the size of the bucket have to  do this? If this is talking about the heuristics of implemention, then that shouldn't be documented.